static/sitemap.xml

```diff
@@ -0,0 +1,222 @@
+  <lastmod>2020-04-08T18:27:10+00:00</lastmod>
+  <priority>0.80</priority>
+</url>
+<url>
+  <loc>https://blog.fleek.co/posts/Guide-IPFS/</loc>
+  <lastmod>2020-04-08T18:27:10+00:00</lastmod>
+  <priority>0.80</priority>
+</url>
+<url>
+  <loc>https://blog.fleek.co/page/1/</loc>
+  <lastmod>2020-04-08T18:27:11+00:00</lastmod>
+  <priority>0.80</priority>
+</url>
+<url>
+  <loc>https://blog.fleek.co/posts/wordpress+fleek/</loc>
+  <lastmod>2020-04-08T18:27:11+00:00</lastmod>
+  <priority>0.64</priority>
+</url>
+<url>
+  <loc>https://blog.fleek.co/posts/fleek-nextJS/</loc>
+  <lastmod>2020-04-08T18:27:11+00:00</lastmod>
+  <priority>0.64</priority>
+</url>
+<url>
+  <loc>https://blog.fleek.co/category/tutorial/page/1/</loc>
+  <lastmod>2020-04-08T18:27:11+00:00</lastmod>
+  <priority>0.64</priority>
+</url>
+<url>
+  <loc>https://blog.fleek.co/tag/tutorial/</loc>
+  <lastmod>2020-04-08T18:27:12+00:00</lastmod>
+  <priority>0.64</priority>
+</url>
+<url>
+  <loc>https://blog.fleek.co/tag/resource/</loc>
+  <lastmod>2020-04-08T18:27:12+00:00</lastmod>
+  <priority>0.64</priority>
+</url>
+<url>
+  <loc>https://blog.fleek.co/tag/announcement/</loc>
+  <lastmod>2020-04-08T18:27:12+00:00</lastmod>
+  <priority>0.64</priority>
+</url>
+<url>
+  <loc>https://blog.fleek.co/tag/release-update/</loc>
+  <lastmod>2020-04-08T18:27:12+00:00</lastmod>
+  <priority>0.64</priority>
+</url>
+<url>
+  <loc>https://blog.fleek.co/tag/rebrand/</loc>
+  <lastmod>2020-04-08T18:27:12+00:00</lastmod>
+  <priority>0.64</priority>
+</url>
+<url>
+  <loc>https://blog.fleek.co/tag/guide/</loc>
+  <lastmod>2020-04-08T18:27:12+00:00</lastmod>
+  <priority>0.64</priority>
+</url>
+<url>
+  <loc>https://blog.fleek.co/tag/ipfs/</loc>
+  <lastmod>2020-04-08T18:27:12+00:00</lastmod>
+  <priority>0.64</priority>
+</url>
+<url>
+  <loc>https://blog.fleek.co/category/guide/</loc>
+  <lastmod>2020-04-08T18:27:12+00:00</lastmod>
+  <priority>0.64</priority>
+</url>
+<url>
+  <loc>https://blog.fleek.co/posts/Gatsby-Fleek/</loc>
+  <lastmod>2020-04-08T18:27:13+00:00</lastmod>
+  <priority>0.64</priority>
+</url>
+<url>
+  <loc>https://blog.fleek.co/category/release/</loc>
+  <lastmod>2020-04-08T18:27:13+00:00</lastmod>
+  <priority>0.64</priority>
+</url>
+<url>
+  <loc>https://blog.fleek.co/posts/release+update+3+17+20/</loc>
+  <lastmod>2020-04-08T18:27:13+00:00</lastmod>
+  <priority>0.64</priority>
+</url>
+<url>
+  <loc>https://blog.fleek.co/page/2/</loc>
+  <lastmod>2020-04-08T18:27:13+00:00</lastmod>
+  <priority>0.64</priority>
+</url>
+<url>
+  <loc>https://blog.fleek.co/tag/wordpress/</loc>
+  <lastmod>2020-04-08T18:27:13+00:00</lastmod>
+  <priority>0.51</priority>
+</url>
+<url>
+  <loc>https://blog.fleek.co/posts/fleek-create-react-app/</loc>
+  <lastmod>2020-04-08T18:27:13+00:00</lastmod>
+  <priority>0.51</priority>
+</url>
+<url>
+  <loc>https://blog.fleek.co/posts/fleek+helloWorld/</loc>
+  <lastmod>2020-04-08T18:27:13+00:00</lastmod>
+  <priority>0.51</priority>
```

Comment:
can we make this maybe the highest priority?

Reply:
change in 0099621